doc/manual/src/glossary.md

```diff
@@ -1,5 +1,15 @@
 # Glossary
 
+  - [installable]{#gloss-installable}\
+    Argument type that can be passed to [`nix-build`](./command-ref/nix-build.md) or [`nix build`](./command-ref/new-cli/nix3-build.md) (experimental):
```

Comment:
```suggestion
    Argument type that can be passed to [`nix build`](./command-ref/new-cli/nix3-build.md):
```

Reply:
The entry should try to give some intuition about what the intent of "installable" is. E.g. from the `nix` manpage:

> Many nix subcommands operate on one or more *installables*. These are command line arguments that represent something that can be built in the Nix store.

Reply:
We should probably also rename them (in a follow-up PR) as they really are not about installing at all.

Once we have https://github.com/NixOS/nix/pull/4543 we can document this better as derived paths are the *ur installable*, everything else is just a more elaborate way of specifying a derived path (with some commands doing things with those extra details).

Reply:
@Edolstra why does the concept not apply to `nix-build`?

Reply:
I mean, it's just not a concept in the old CLI (in the source code, error messages, docs...). Obviously the old CLI has a similar overloading of command line arguments, but different (e.g. `nix-build foo.nix -A bar` vs `nix build -f foo.nix bar` - so we can't say that `foo.nix` is an installable).

Reply:
@fricklerhandwerk yeah fwiw the code in the implemementation called `Installable` isn't used by the old CLI either. The old CLI is quite ad-hoc and doesn't have these reusable concepts in its implementation.